Source/Global/perform_env.cpp

```diff
@@ -260,7 +356,7 @@ HRESULT CALLBACK HC_PERFORM_ENV::CleanupAsyncProvider(XAsyncOp op, const XAsyncP
 
         RETURN_IF_FAILED(CurlProvider::CleanupAsync(std::move(curlProvider), curlCleanupAsyncBlock.get()));
```

Comment:
github is dumb and doesn't let me put this comment where I want

I would define reclaim as the very first thing so that we can freely early out in the code

(note that this code can throw on line 337 for example)